Makefile

```diff
@@ -14,30 +14,18 @@ BUILD=$(TOOLS)/build.py
 NORMALIZE=$(TOOLS)/sfdnormalize.py
 CHECKERRS=$(TOOLS)/check-errors.py
 
-NULL=
-
-FONTS=Sans-Regular \
-       Sans-Bold \
-       Sans-Italic \
-       Serif-Regular \
-       Serif-Semibold \
-       Serif-Bold \
-       Serif-Italic \
-       Serif-SemiboldItalic \
-       Serif-BoldItalic \
-       SerifDisplay-Regular \
-       Math-Regular \
-       SerifInitials-Regular \
-       Mono-Regular \
-       Keyboard-Regular \
-       $(NULL)
-
-SFD=$(FONTS:%=$(SOURCEDIR)/$(NAME)%.sfd)
-NRM=$(FONTS:%=$(BUILDDIR)/$(NAME)%.nrm)
-CHK=$(FONTS:%=$(BUILDDIR)/$(NAME)%.chk)
-DUP=$(FONTS:%=$(BUILDDIR)/$(NAME)%.dup)
-LNT=$(FONTS:%=$(BUILDDIR)/$(NAME)%.lnt)
-OTF=$(FONTS:%=$(NAME)%.otf)
+# Discover all font source files
+SFD=$(wildcard $(SOURCEDIR)/*.sfd)
```

Comment:
Actually, I prefer listing the font files explicitly. I don’t want some random SFD file that happen to be in the sources directory to be built and added to the release tarball.

Reply:
Okay, I can see the logic in that. It makes it harder to hack on, but also harder to break. It also has the advantage of being able to manually specify an order, which will be useful elsewhere. I'll re-instate a manual control over that.